docs: Add 1.3.0 release note (#447)
decred/vspd: docs/release-notes/release-notes.1.3.0.md

## Recommended Upgrade Path

The upgrade path below includes vspd downtime, during which clients will not be
able to register new tickets, check their ticket status, or update their voting
preferences. Voting on tickets already registered with the VSP will not be
interrupted. You may wish to put up a temporary maintenance webpage or announce
downtime in public channels.

The upgrade path assumes dcrd and dcrwallet are already version 1.8.0.

1. Build vspd from the 1.3.0 release tag.
1. Stop vspd.
1. **Make a backup of the vspd database file in case rollback is required.**
1. Install new vspd binary and start it.
1. Check vspd log file for warnings or errors.
1. Log in to the admin webpage and check the VSP Status tab for any issues.

## Notable Changes

- Fee calculation now takes the new block reward subsidy split from the activation
  of [DCP-0012](https://github.com/decred/dcps/blob/master/dcp-0012/dcp-0012.mediawiki)
  into consideration. In practice, this means that VSPs will begin charging
  marginally higher fees.
- vspd will now distinguish between tickets which are "missed" and tickets which
  are "expired". Previously these tickets would be considered as a single set

>   labelled "expired". This is acheived using dcrd and

Reviewer: Not sure if you aim for American or British English, but one l for American.  Doesn't really matter, but probably best to be consistent with whichever style is chosen.

preferences. Voting on tickets already registered with the VSP will not be
interrupted. You may wish to put up a temporary maintenance webpage or announce
downtime in public channels.

The upgrade path assumes dcrd and dcrwallet are already version 1.8.0.

1. Build vspd from the 1.3.0 release tag.
1. Stop vspd.
1. **Make a backup of the vspd database file in case rollback is required.**
1. Install new vspd binary and start it.
1. Check vspd log file for warnings or errors.
1. Log in to the admin webpage and check the VSP Status tab for any issues.

## Notable Changes

- Fee calculation now takes the new block reward subsidy split from the activation
  of [DCP-0012](https://github.com/decred/dcps/blob/master/dcp-0012/dcp-0012.mediawiki)
  into consideration. In practice, this means that VSPs will begin charging
  marginally higher fees.
- vspd will now distinguish between tickets which are "missed" and tickets which
  are "expired". Previously these tickets would be considered as a single set
  labelled "expired". This is acheived using dcrd and
  [Golomb-Coded Set filters](https://github.com/decred/dcrd/tree/master/gcs#gcs).

>   vspd 1.3.0 will perform a **one-time update of every revoked ticket in the

Reviewer: ```suggestion
-  vspd 1.3.0 will perform a **one-time update of every revoked ticket in the
```

  database** the first time it is started. This may take a while for VSPs which
  have been active for a long time or have a large number of revoked tickets.
- Expired and missed tickets added to `/vspinfo`. Revoked is now deprecated.
- Minor improvements to web UI:
  - Homepage now displays expired and missed tickets instead of revoked tickets.

>   - Homepage no longer displays the current network unless it is running on

Reviewer: ```suggestion
  - Homepage no longer displays the current network unless it is not running mainnet
```

    testnet or simnet.
  - Admin page now displays decoded transactions in a human readable format

>     rather than just the raw bytes.

Reviewer: ```suggestion
    in addition to just the raw bytes.
```
